overrides/ios-override.json

```diff
@@ -170,6 +170,9 @@
                             }
                         ]
                     }
+                },
+                "filterlist": {
+                    "state": "disabled"
```

Comment:
Will update to add `minSupportedVersion` and flip to enabled once it's confirmed which version this is going out in.